package.json

```diff
@@ -30,7 +30,7 @@
     "coverage": "jest --coverage"
   },
   "dependencies": {
-    "@ensembl/ensembl-genome-browser": "0.2.1",
+    "@ensembl/ensembl-genome-browser": "0.3.0-test",
```

Comment:
**NOTE TO SELF**: Update this to `0.3.0` before merging

Reply:
done